_posts/2021/2021-05-31-feathers-js-overview.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: FeathersJS - A brief introduction
+date: 2021-06-10
+header_image: public/feathers-cover.png
+header_position: center
+header_overlay: true
+category: methods-and-tools
+tags: ["api", "express.js", "javascript", "fullstack"]
+authors: ["Peter B."]
+about_authors: ["pboxall"]
+---
+
+FeathersJS is a web-framework for creating realtime applications as well as REST APIs in JavaScript or Typescript.
+It abstracts away the needed database logic for using different databases and eases the creation of real-time applications such as WhatsApp or Telegram.
+
+## A short history of full stack JavaScript
+
+Back in the day before the big environment around it was build, JavaScript was a language mainly used for scripts that run directly on web pages or for the creation of client applications loaded in the browser (Single Page Apps - like Google Maps).
+For the backend a much more reputable and sophisticated language would be used like Java/C#(DotNet).
+
+But things have changed over the years. Now, JavaScript can be used across the full stack for different applications.
+This means, for example, a Model View Controller (MVC) web application wouldn't only require familiarity with this one language from the developer but also the ability to develop on all parts of the application.
```

Comment:
Maybe add a link on "Model View Controller (MVC) web application" to a website with more information about this topic or add more here.